sourcecode/apis/contentauthor/app/Libraries/H5P/H5PConfigAbstract.php

```diff
@@ -14,11 +14,11 @@
 
 abstract class H5PConfigAbstract implements ConfigInterface
 {
-    protected const CACHE_BUSTER_STRING = '2.0.3';
+    public const CACHE_BUSTER_STRING = '2.0.3';
     protected const EMBED_TEMPLATE = '<iframe src="%s" width=":w" height=":h" frameborder="0" allowfullscreen="allowfullscreen" allow="geolocation *; microphone *; camera *; midi *; encrypted-media *" title="%s"></iframe>';
 
     public function __construct(
-        protected H5PAdapterInterface $adapter,
+        public H5PAdapterInterface $adapter,
```

Comment:
Why the change to public?

Reply:
Used it in the tests to verify that the correct Adapter was set, and forgot to revert it